### ISSUE TITLE
Patch release 

### ISSUE THREAD

**Issue by mts299** [SuperDARN/pydarn#177]
# Release

**Release number**: v2.1.1

- [ ] Major - Structural/User interface change
- [ ] Minor - New feature or Deprecated feature
- [x] Patch - bug fixes 

## Scope

Patch release focused around these bugs that have popped up with regards to RTP and Fan plots. 

## Propose Deadline: 

**Date**: July 30th 

## Pull Requests

**All PRs merged into `develop` will be included in this release**

### Pull Request To Be Included

- [x] #172 
- [x] #174  - PR ready for testing
- [x] #175 - PR ready for testing
- [x] #167 - waiting on @carleyjmartin to make a PR?

**Comment by carleyjmartin** [comment]
#167 will need the kwargs pr to be merged too as the solution is based off that (#169)

**Comment by mts299** [comment]
O whopps okay we can probably merge kwargs if there are no conflicts with the new merges :p

**Comment by carleyjmartin** [comment]
The start time PR is now available to test at #178 :D

**Comment by carleyjmartin** [comment]
As a side comment too: when I merged develop into the above new PR I found that lines 721-723 of rtp.py - 
```python
lines = ax.plot_date(x, my, fmt='k', tz=None, xdate=True,
                                 ydate=False, color=color, linestyle=linestyle,
                                 linewidth=linewidth)
```
- threw a User Warning about setting `color=color` when plotting only a time series plot (assuming it's suppressed when plotting summary?) Doesn't effect the actual plotting at all though. 
```
722: UserWarning: color is redundantly defined by the 'color' keyword argument and the fmt string "k" (-> color=(0.0, 0.0, 0.0, 1)). The keyword argument will take precedence.
```

**Comment by mts299** [comment]
@carleyjmartin did you fix this in your PR or shall I make one as a fix?

**Comment by carleyjmartin** [comment]
No I didn't, I tried, I wasn't sure how tbh.
